src/libs/OptionsListUtils.js

```diff
@@ -38,9 +38,9 @@ Onyx.connect({
 const policies = {};
 Onyx.connect({
     key: ONYXKEYS.COLLECTION.POLICY,
-    callback: (policy) => {
-        if (policy && policy.id) {
-            policies[policy.id] = policy;
+    callback: (policy, key) => {
+        if (policy && key && policy.name) {
```

Comment:
Just a note for why I needed to be specific about `policy.name` here being included because otherwise when [this function](https://github.com/Expensify/Expensify.cash/blob/099457ec41f238be8103d953e4c8b50ea2b030f1/src/libs/actions/Policy.js#L81-L92) is called from here:
https://github.com/Expensify/Expensify.cash/blob/8a221022b633404eb84960b23b64ca884dc51fc4/src/libs/Navigation/AppNavigator/AuthScreens.js#L143

we end up calling the Onyx callback here and overwriting the policies with just their employeesList. It'll overwrite all the policy objects in `policies` with just their `employeeList`. I believe a race condition is causing this. By checking for `policy.name`, here we won't overwrite and delete the policy names here.

To see this in effect, you can take out `&& policy.name` and see that the search bar will always show Unknown Policy when you open it:
![image](https://user-images.githubusercontent.com/19364431/124050283-4187f000-d9e8-11eb-9369-8ee3b2aad283.png)